capsules/extra/src/rf233.rs

```diff
@@ -1346,6 +1346,8 @@ impl<'a, S: spi::SpiMasterDevice<'a>> radio::RadioData<'a> for RF233<'a, S> {
         self.rx_buf.replace(buffer);
     }
 
+    fn set_ack_buffer(&self, _buffer: &'static mut [u8]) {}
+
```

Comment:
```suggestion
```

Ideally, fixing nrf52840-specific stuff wouldn't need to touch other radio implementation